### PR TITLE
api/server: add --no-offload option

### PR DESCRIPTION
**Pull request by outscale-mgo**
Signed-off-by: Matthias Gatto <matthias.gatto@outscale.com>

### CODE REVIEW

File: tests/functions.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ function ssh_run_background {
     id=$1
     cmd="${@:2}"
     key=$BUTTERFLY_BUILD_ROOT/vm.rsa
-    ssh -q -f -p 500$id -l root -i $key -oStrictHostKeyChecking=no 127.0.0.1 $cmd &> /dev/null
+    ssh -q -f -p 500$id -l root -i $key -oStrictHostKeyChecking=no 127.0.0.1 $cmd > /dev/null
```
Comment (outscale-jju) [inline tests/functions.sh]: why showing ssh details ?

**Comment by outscale-jju** [comment]
typo in commit message "\test: ..."

**Comment by outscale-jju** [comment]
tabs found in scripts ;)

File: tests/functions.sh (hunk 1 above)
Comment (outscale-mgo) [inline tests/functions.sh]: it's not showing detail, but error :p 
but I guess that's unrealted

**Pull request merged**